Sources/Overload/OvTools/src/OvTools/Time/Date.cpp

```diff
@@ -3,28 +3,26 @@
 * @author: Overload Tech.
 * @licence: MIT
 */
+#define _CRT_SECURE_NO_WARNINGS // use of localtime()
 
 #include <ctime>
 
 #include "OvTools/Time/Date.h"
 
-std::string OvTools::Time::Date::GetDateAsString()
-{
-	std::string date;
-	const auto now = time(nullptr);
-	tm ltm;
-
-	localtime_s(&ltm, &now);
-
-	std::string dateData[6] =
-	{
-		std::to_string(1900 + ltm.tm_year),
-		std::to_string(1 + ltm.tm_mon),
-		std::to_string(ltm.tm_mday),
-		std::to_string(ltm.tm_hour),
-		std::to_string(ltm.tm_min),
-		std::to_string(ltm.tm_sec)
-	};
+std::string OvTools::Time::Date::GetDateAsString() {
+    std::string date;
+    const auto now = time(nullptr);
+    tm ltm = *localtime(&now);
```

Comment:
I'm not keen on `localtime` usage here. It isn't safe, and `localtime_s` is prefered